lib/API/JSON/v1/Settings.php

```diff
@@ -30,22 +35,41 @@ class Settings extends APIEndpoint {
   /** @var ServicesChecker */
   private $servicesChecker;
 
+  /** @var WPFunctions */
+  private $wp;
+
   public $permissions = [
     'global' => AccessControl::PERMISSION_MANAGE_SETTINGS,
   ];
 
+  /**
```

Comment:
I noticed that you mixed one-line annotations and multi-line annotations. It's a detail only.